mlflow/protos/facet_feature_statistics.proto

```diff
@@ -0,0 +1,329 @@
+syntax = "proto2";
```

Comment:
Can we add a comment to link explicitly to the GH commit to this file we copy from Facet? It'd be helpful to debug in the future if Facet introduces some backward in compatible changes.